DQM/SiPixelPhase1Summary/src/SiPixelPhase1Summary.cc

```diff
@@ -311,6 +304,23 @@ void SiPixelPhase1Summary::fillTrendPlots(DQMStore::IBooker & iBooker, DQMStore:
     }
   }
 
+  if (!runOnEndLumi_) return; // The following only occurs in the online
+  //Reset some MEs every 10LS here
+  for (auto it : {1,2,3,4}) { //PXBarrel
```

Comment:
'toReset' is the name of the MonitorElement variable that we're looking to reset - the if (toReset) condition is to make sure that the ME exists before trying to reset it (to prevent seg faults). I guess the name is kind of confusing, should I change that?